### ISSUE TITLE
improve package visibility

### ISSUE THREAD

**Issue by cdaringe** [node-swig/swig-templates#30]
it's hard to find this package! googling it, even with a bunch of keywords, still gets you to paul's original project.

- [ ] update paularmstrong's docs to point to our docs
- [ ] update `swig` npm packages to point to our community package

i know paul has demonstrated some reluctance to let go of the reigns.  getting his help here would be a big step in realization that the project is alive and kickin'

**Comment by ecoleman** [comment]
It might be a good idea to change the install instructions on http://node-swig.github.io/swig-templates/ until this is resolved...

**Comment by nrambeck** [comment]
I thought this project was completely abandoned and was looking for other alternatives until I checked one more time and followed the original repo's fork history. Definitely need to have a better way for people to find this new community project.

**Comment by cdaringe** [comment]
agreed to both @ecoleman @nrambeck.  if either of ya wants to start any effort on the matter, it would be gladly welcomed :)

**Comment by ecoleman** [comment]
Has anyone talked with Paul? I can reach out to him and check NPM rules for abandoned packages...

**Comment by kgarsjo** [comment]
For reference, here's the NPM [disputes page](https://docs.npmjs.com/misc/disputes).

**Comment by cdaringe** [comment]
we're probably getting ahead of ourselves with that.  no, i don't think anyone has reached out recently.  the 3-4 of us who have done some continued work in this project haven't gone to great lengths to re-request name rights.

in regards to @ecoleman's remark about updating the docs, great point.  filing it as a separate issue, #32

**Comment by elporfirio** [comment]
Well finally im here, so we need create content with swig-templates instead "swig" alone, ill try todo some tutorials :D for SEO tracking

**Comment by migg24** [comment]
First thing you should do is make clear that there are no breaking changes in updating from swig 1.4.2 to swig-templates 2.0.2. This is not made clear anywhere, I can only guess this is the case from the small comment in the CHANGELOG.

If this is confirmed you should definitely add this as the first thing on the README to show everybody that they can just switch the packagein their dependencies.
